### PR TITLE
Various typos and wording fixes

### PR DESCRIPTION
**Pull request by david4096**
* fastq capitalized to FASTQ
* Fixed some spelling errors
* Two slight rewordings for readability

### CODE REVIEW

File: doc/source/api/metadata.rst [hunk 1]
```diff
@@ -110,13 +110,13 @@ Durations
 =========
 
 
-Durations are the most common form of time intervals. They do not reference to
+Durations are the most common form of time intervals. They do not reference make to
```
Comment (jmarshall) [inline doc/source/api/metadata.rst]: Huh?

Comment (mbaudis) [inline doc/source/api/metadata.rst]: Think this was intense to read "... They do not make reference to ..." (though I think the original was o.k.?).

To emphasise the difference, the next sub-header could be changed to "(Anchored) Time Intervals" or such?

**Comment by mbaudis** [comment]
+1 when fixing 113 in doc/source/api/metadata.rst

**Comment by david4096** [comment]
Thanks @jmarshall, I did mean to add "make reference". I changed the wording from "reference" to "refer".

```
-Durations are the most common form of time intervals. They do not reference to
 +Durations are the most common form of time intervals. They do not refer to
```

**Pull request merged**